druid/src/env.rs

```diff
@@ -208,6 +208,43 @@ impl Env {
             .map(|value| value.to_inner_unchecked())
     }
 
+    /// Gets a value from the environment, in its encapsulated [`Value`] form,
+    /// expecting the key to be present.
+    ///
+    /// *WARNING:* This is not intended for general use, but only for inspecting an `Env` e.g.
+    /// for debugging, theme editing, and theme loading.
+    ///
+    /// # Panics
+    ///
+    /// Panics if the key is not found
+    /// [`Value`]: enum.Value.html
+    pub fn get_untyped(&self, key: impl Borrow<Key<()>>) -> &Value {
+        let key = key.borrow();
+        if let Some(value) = self.0.map.get(key.key) {
+            value
+        } else {
+            panic!("key for {} not found", key.key)
+        }
+    }
+
+    /// Gets a value from the environment, in its encapsulated [`Value`] form,
+    /// returning None if a value isn't found.
+    ///
+    /// *WARNING:* This is not intended for general use, but only for inspecting an `Env` e.g.
+    /// for debugging, theme editing, and theme loading.
+    /// [`Value`]: enum.Value.html
+    pub fn try_get_untyped(&self, key: impl Borrow<Key<()>>) -> Option<&Value> {
+        self.0.map.get(key.borrow().key)
+    }
+
+    /// Gets the entire contents of the `Env`, in key-value pairs.
+    ///
+    /// *WARNING:* This is not intended for general use, but only for inspecting an `Env` e.g.
+    /// for debugging, theme editing, and theme loading.
+    pub fn get_all(&self) -> impl ExactSizeIterator<Item = (&String, &Value)> {
```

Comment:
a little late to the game but I might've just called this `iter` or `iter_raw` or something. 🤷

Reply:
Ah. Hm. That might have made some idiomatic sense. No reason not to change it. I'll try to find some time this weekend.

Reply:
no pressure, just an observation!